tests/pdarray_creation_test.py

```diff
@@ -62,23 +61,34 @@ def test_bigint_creation(self):
             self.assertIsInstance(pda, ak.pdarray)
             self.assertEqual(10, len(pda))
             self.assertEqual(ak.bigint, pda.dtype)
+            self.assertEqual(pda[-1], bi + 9)
 
         # test array and arange infer dtype
         arr = ak.array([bi, bi + 1, bi + 2, bi + 3, bi + 4])
         ara = ak.arange(bi, bi + 5)
+        # TODO add .to_list and .to_ndarray functionality
```

Comment:
Do we have an issue for this? I am guessing it is not supported for `dtype=bigint` since NumPy sets the type as object.

Reply:
it's being tracked in the `Add BigInt support` parent issue #1919

What you're saying is true, but I think we can special case it and just check for `dtype == bigint` and then select object. What really makes this difficult is `to_ndarray` uses `bytesToSymEntry` and it's not obvious to me that this would play nice with chapel `bigint` type. I'm planning on looping in elliot when I work this

https://github.com/Bears-R-Us/arkouda/blob/master/src/GenSymIO.chpl#L53-L60

Reply:
Sweet thanks!